md/bdm-management-in-bonita-bpm-portal.md

```diff
@@ -35,4 +35,25 @@ in the business database, so as to store business objects appropriately when mod
 :::
 
 It is recommended to perform a backup of the business database before deploying a business data model.
-(If the deployment fails, check the engine logs to analyse the problem.)
+(If the deployment fails, check the engine logs to analyse the problem.)  
+
+::: info
+**Note:** For Performance and Efficiency editions only.
+:::
+
+It is possible to define Business Data Model access control rules in Bonita Studio and import them in Bonita Portal.  
+The Access control rules must match the Business Data model.
+
+1. To import the BDM access control rules created in Bonita Studio, first [export the Access Control file](bdm-access-control.md) from Bonita Studio where it was designed.
+2. In the Bonita Portal, log in as the technical user (default login 'install', default password 'install') or as a user with the administrator profile.
+3. As the technical user, go to the **Business Data Model** menu and choose **Access Control**. As an administrator, go to the **BDM** menu.
+4. A page with a section **Import a BDM access control file** is displayed.
+5. Choose the file containing the Business Data Model exported from the Bonita Studio, and click on _**Import**_.
```

Comment:
Choose the file containing the Business Data Model access control definition exported from the Bonita Studio..